src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -73,15 +76,7 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs e)
                     var oldDocument = e.OldSolution.GetDocument(e.DocumentId);
                     var newDocument = e.NewSolution.GetDocument(e.DocumentId);
 
-                    var oldTextVersion = oldDocument.GetTextVersionAsync(CancellationToken.None).WaitAndGetResult(CancellationToken.None);
-                    var newTextVersion = newDocument.GetTextVersionAsync(CancellationToken.None).WaitAndGetResult(CancellationToken.None);
-
-                    if (oldTextVersion != newTextVersion)
-                    {
-                        UpdateClassVersion();
-                        UpdateMembersVersion();
-                    }
-
+                    _documentChangedWorker.ContinueWith(_ => DocumentChangedAsync(oldDocument, newDocument), TaskScheduler.Default);
```

Comment:
Oh, no, this is worse - this is an async method in a non-async lambda.  I meant make `OnWorkspaceChanged` into an `async` method instead, if the work done doesn't actually justify using a threadpool thread for `OnDocumentChanged`.